PopPUNK/network.py

```diff
@@ -67,29 +67,54 @@ def fetchNetwork(network_dir, model, refList, ref_graph = False,
     """
     # If a refined fit, may use just core or accessory distances
     dir_prefix = network_dir + "/" + os.path.basename(network_dir)
+    if use_gpu:
+
+        # load CUDA libraries
+        try:
+            import cugraph
+            import cudf
+        except ImportError as e:
+            sys.stderr.write("cugraph and cudf unavailable\n")
+            raise ImportError(e)
```

Comment:
Similar to in models, it would be better if this was just in one place at the top of the file

Reply:
Done in e18ed35.